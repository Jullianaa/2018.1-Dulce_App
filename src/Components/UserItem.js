import React from 'react';
import {Text, TouchableHighlight} from 'react-native';

const styles = {
  item: {
    height: 60,
    borderBottomWidth: 1,
    justifyContent: 'center'
  },
  text: {
    color: 'black',
    fontSize: 25
  }
};

class UserItem extends React.Component {
  render() {
    return (
      <TouchableHighlight
        onPress={this.props.onPress}
        style={styles.item}
      >
        <Text style={styles.text}>{this.props.text}</Text>
      </TouchableHighlight>
    );
  }
}

export default UserItem;
