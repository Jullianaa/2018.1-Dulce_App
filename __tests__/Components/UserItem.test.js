import React from 'react';
import UserItem from '../../src/Components/UserItem';
import Adapter from 'enzyme-adapter-react-16';
import {shallow} from 'enzyme';
import Enzyme from 'enzyme';
import renderer from 'react-test-renderer';
require('bezier');

Enzyme.configure({adapter: new Adapter()});

it('renders correctly', () => {
  const tree = renderer
  .create(<UserItem />)
  .toJSON();
  expect(tree).toMatchSnapshot();
});
