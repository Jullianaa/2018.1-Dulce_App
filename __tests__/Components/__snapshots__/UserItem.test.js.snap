// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  hasTVPreferredFocus={undefined}
  hitSlop={undefined}
  isTVSelectable={true}
  nativeID={undefined}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "borderBottomWidth": 1,
      "height": 60,
      "justifyContent": "center",
    }
  }
  testID={undefined}
  tvParallaxProperties={undefined}
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "black",
        "fontSize": 25,
      }
    }
  />
</View>
`;
